src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/AssemblyBuilder.cs

```diff
@@ -63,17 +63,6 @@ public sealed partial class AssemblyBuilder : Assembly
             // Clone the name in case the caller modifies it underneath us.
             name = (AssemblyName)name.Clone();
 
-            // Scan the assembly level attributes for any attributes which modify how we create the
```

Comment:
Did this change semantics?

Previously a `null` `unsafeAssemblyAttributes` would cause the attribute list to be created here and populated later on below. Now the `null` won't cause the list to be created here and thus isn't populated later on below.

Reply:
I do not think that this is changing any observable behavior.

The original code in .NET Framework code had to create a defensive copy for partial trust sandbox security: https://github.com/microsoft/referencesource/blob/5697c29004a34d80acdaf5742d7e699022c64ecd/mscorlib/system/reflection/emit/assemblybuilder.cs#L370-L422

We do not have partial trust sandbox security in .NET Core and so we do not need this defensive copy anymore.